packages/stacking/src/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { AddressHashMode } from '@stacks/transactions';
+import { address } from 'bitcoinjs-lib';
+import BN from 'bn.js';
+
+export function getAddressHashMode(btcAddress: string) {
+  if (btcAddress.startsWith('bc1') || btcAddress.startsWith('tb1')) {
+    const { data } = address.fromBech32(btcAddress);
+    if (data.length === 32) {
+      return AddressHashMode.SerializeP2WSH;
+    } else {
+      return AddressHashMode.SerializeP2WPKH;
+    }
+  } else {
+    const { version } = address.fromBase58Check(btcAddress);
+    switch (version) {
+      case 0:
+        return AddressHashMode.SerializeP2PKH;
+      case 111:
+        return AddressHashMode.SerializeP2PKH;
+      case 5:
+        return AddressHashMode.SerializeP2SH;
+      case 196:
+        return AddressHashMode.SerializeP2SH;
+      default: 
+        throw new Error('Invalid pox address version');
+    }
+  }
+}
+
+export function convertBTCAddress(btcAddress: string) {
```

Comment:
`convertBtcAddress` ?

Reply:
Had to use this function recently for testing in the sidecar. The current name is meaningless, should be something like `decodeBtcAddress` or `destructureBtcAddress`

Reply:
Changed to `decodeBtcAddress`